packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.html

```diff
@@ -1,7 +1,16 @@
 <div data-ovh-alert></div>
 
-<h1 data-translate="dedicated_server_interfaces_title"></h1>
+<div class="row">
+    <h1 data-translate="dedicated_server_interfaces_title" class="col-auto mr-auto"></h1>
```

Comment:
`.row`, the child sould be a `div.col-*`

Reply:
@jleveugle is already the case (see: https://getbootstrap.com/docs/4.3/layout/grid/#margin-utilities)

Reply:
It comes from bs4 doc : 

```
  <div class="row">
    <div class="col-auto mr-auto">.col-auto .mr-auto</div>
    <div class="col-auto">.col-auto</div>
  </div>
```